client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx

```diff
@@ -0,0 +1,131 @@
+
+import Toggle from 'commons/Toggle/Toggle';
+import DatePick from 'commons/DatePick/DatePick';
+import InteractionEventVariables from 'models/InteractionEventVariables';
+
+import useStyles from './NewInteractionEventDialogStyles';
+import PrimaryButton from 'commons/Buttons/PrimaryButton/PrimaryButton';
+
+export interface Props {
+    isOpen: boolean,
+    eventId: number,
+    onCancle: (eventId: number) => void;
+    onCreateEvent: (interactionEventVariables: InteractionEventVariables) => void;
+}
+
+const selectData = [
+    'מקום 1',
+    'מקום 2',
+    'מקום 3',
+    'מקום 4',
+    'מקום 5'
+]
+
+const defaultTime : string = '00:00';
+
+const newConactEventTitle = 'יצירת מקום/מגע חדש'
+
+const NewInteractionEventDialog : React.FC<Props> = (props: Props) : JSX.Element => {
+    
+    const { isOpen, eventId, onCancle, onCreateEvent } = props;
+
+    const classes = useStyles({});
+    const [canCreateEvent, setCanCreateEvent] = React.useState<boolean>(false);
+    const [placeType, setPlaceType] = React.useState<string>(selectData[0]);
+    const [eventStartTime, setEventStartTime] = React.useState<string>();
+    const [eventEndTime, setEventEndTime] = React.useState<string>();
+    const [canBeExported, setCanBeExported] = React.useState<boolean>(false);
+
+    React.useEffect(() => {
+        setCanCreateEvent(eventStartTime !== undefined && eventEndTime !== undefined);
+    }, [eventStartTime, eventEndTime]);
+
+    return (
+        <Dialog classes={{paper: classes.dialogPaper}} open={isOpen} maxWidth={false}>
+            <DialogTitle className={classes.dialogTitleWrapper}>
+                {newConactEventTitle}
+            </DialogTitle>
+            <DialogContent>
+                <Grid className={classes.form} container justify='flex-start'>
+                    <div className={classes.rowDiv}>
+                        <Grid item xs={3}>
+                            <Typography variant='caption' className={classes.fieldName}>
+                                סוג אתר:
+                            </Typography>
+                        </Grid>
+                        <Grid item xs={9}>
+                            <Select
+                                value={placeType}
+                                onChange={(event: React.ChangeEvent<any>) => setPlaceType(event.target.value)}
+                                className={classes.placeTypeSelect}
+                            >
+                                {
+                                    selectData.map((placeName: string) => (
+                                        <MenuItem key={placeName} value={placeName}>{placeName}</MenuItem>
+                                    ))
+                                }
+                            </Select>
+                        </Grid>
+                    </div>
+                    <div className={classes.rowDiv}>
```

Comment:
maybe put the className in const or export it? because it can be used a lot throughout the project.
the same with fieldName and form maybe

Reply:
I created a file that contains all the common form styles under styles folder

Reply:
yay!